.cloudbees/testing/action.yml

```diff
@@ -84,6 +84,10 @@ inputs:
   gitlab-server-url:
     description: The base URL for the GitLab instance that you are trying to clone from, will use environment defaults (i.e. the GITLAB_SERVER_URL environment variable) to fetch from the same instance that the workflow is running from unless specified. Example URLs are https://gitlab.com or https://my-gl-server.example.com
     required: false
+  create-tarfile:
```

Comment:
this needs to be kept in sync with the root action.yaml. Why am I not seeing the matching change there?

Reply:
I thought it should be updated since I added a new flag for the checkout action.

Reply:
you have to add the flag in both places